nixos/lib/default.nix

```
@@ -29,10 +32,23 @@ in
   using a binding like `nixosLib = import (nixpkgs + "/nixos/lib") { }`.
 */
 {
+
+  # experimental
+
   inherit (seqAttrsIf (!featureFlags?minimalModules) minimalModulesWarning eval-config-minimal)
     evalModules
     ;
 
+  inherit (seqAttrsIf (!featureFlags?standardModules) standardModulesWarning eval-config-standard)
```

```suggestion
  inherit (seqAttrsIf (!featureFlags?publicEvalSystem) publicEvalSystemWarning eval-config-standard)
```

A bit more meaningful I think.

---

I removed the feature flag straight away... That makes more sense when we start dogfooding it already.